ngx-tools/src/utilities/object-deep-parse.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Helper function to parse an object with deep keys
+ *
+ * @param obj - the object
+ * @param path - the object path
+ * @return string value at the lowest layer
+ */
+
+export function objectDeepParse(object: { [key: string]: any } | string, keys: string | string[], defaultVal = object): any {
```

Comment:
Why does a deep parse function need a default value?

Reply:
We don't need it here. I was actually using lodash `_.get` function call and then wrote its implementation after knowing we're not gonna support it.